tests/resources/opentelemetry/telegraf2.conf

```diff
@@ -0,0 +1,27 @@
+# Centreon telegraf configuration
+# This telegraf configuration is generated by centreon centengine
+[agent]
+  ## Default data collection interval for all inputs
+  interval = "60s"
+
+[[outputs.opentelemetry]]
+   service_address = "127.0.0.1:4317"
+
+
+[[inputs.exec]]
+  name_override = "host_3__service_5"
```

Comment:
Noob question here, is it normal that you have two underscode in the name override ?
should'nt it be host_3_service_5 ?

Reply:
I have forgotten to remove these files

Reply:
removed